libafl_nyx/build.rs

```diff
@@ -7,10 +7,8 @@ fn main() {
         let status = Command::new("./build_nyx_support.sh")
             .status()
             .expect("can't run ./build_nyx_support.sh");
-        if status.success() {
-            println!("success to run ./build_nyx_support.sh");
-        } else {
-            panic!("fail to run ./build_nyx_support.sh");
+        if !status.success() {
+            println!("cargo:warning=fail to run ./build_nyx_support.sh");
```

Comment:
Should we instead ... fix `build_nyx_support`?

Reply:
Or.. should that even be in build.rs or instead in a makefile?

Reply:
it is not necessarily a problem in that shell script
because sometimes it is network issue
sometimes it is git issue